templates/strings/structured-swift4.stencil

```diff
@@ -82,7 +82,13 @@ extension {{enumName}} {
   }
```

Comment:
You're not using the cached bundle in this template.

Reply:
Yea.. I missed a lot places... Use checked with rg and should be good. Pls review again.